x-pack/plugins/observability/public/data/slo/slo.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { cloneDeep } from 'lodash';
-import { v1 as uuidv1 } from 'uuid';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
@kdelemme @simianhacker All the changes for AO are related to SLO. Do you see any issue with this change?

Reply:
Looks good to me.

Reply:
I will give this a proper review.